src/modules/Lan.rb

```diff
@@ -660,7 +661,8 @@ def Packages
         pkgs << "NetworkManager" if !PackageSystem.Installed("NetworkManager")
       elsif !PackageSystem.Installed("wpa_supplicant")
         # we have to add wpa_supplicant when wlan is in game, wicked relies on it
-        pkgs << "wpa_supplicant" if !LanItems.find_type_ifaces("wlan").empty?
+        wlan_present = yast_config.interfaces.any? { |i| i.type == Y2Network::InterfaceType::WIRELESS }
```

Comment:
honestly .. didn't try ... bcs it can be "translated" to ```interfaces.select(type).any?``` ... so using ```interfaces.any?``` seemed more accurate to me (bcs I somehow live in imagination that ```any``` stops on first success ... but I'm not 100% sure if i'm right ;-)